File: app/src/main/java/com/example/cykashoppinglist/mapper/ShoplistItemMapper.java
package com.example.cykashoppinglist.mapper;

import com.example.cykashoppinglist.entity.ShoplistItem;

import org.json.JSONException;
import org.json.JSONObject;

public class ShoplistItemMapper {
	public static ShoplistItem toEntity(JSONObject jsonObject) throws JSONException {
		ShoplistItem item = new ShoplistItem();
		item.setName(jsonObject.getString("content"));
		return item;
	}
}

File: app/src/main/java/com/example/cykashoppinglist/mapper/ShoplistItemMapper.kt
package com.example.cykashoppinglist.mapper

import com.example.cykashoppinglist.entity.ShoplistItem
import org.json.JSONException
import org.json.JSONObject

object ShoplistItemMapper {
	@Throws(JSONException::class)
	fun toEntity(jsonObject: JSONObject): ShoplistItem {
		val item = ShoplistItem()
		item.name = jsonObject.getString("content")
		return item
	}
}